tests/test_basic/test_basic.py

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env python
 
+import os
 import sys
-sys.path.insert(0, '..')
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, os.pardir))
```

Comment:
`openmc` is imported in in input_set which is imported in testing_harness... I just realized that probably the better way to handle this would have been to add `../..` to the path in input_set.py.  Or maybe we should just set PYTHONPATH in run_tests.py?